galite-core/src/main/kotlin/org/kopi/galite/chart/VDefaultChartActor.kt

```diff
@@ -18,5 +18,33 @@
 
 package org.kopi.galite.chart
 
-class VDefaultChartActor {
+import org.kopi.galite.visual.VActor
+import org.kopi.galite.visual.VlibProperties
+
+class VDefaultChartActor(menuIdent: String,
+                         actorIdent: String,
+                         iconName: String,
```

Comment:
I dont thnik so. because this way we will add `iconName` property to the base class ( `VDefaultChartActor` )
`this.iconName` is the `iconName` in super class
Base classes dont have `iconName` property